src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -92,11 +92,21 @@ public BillableUsage processBillableUsage(BillingWindow billingWindow, BillableU
     return toBill;
   }
 
-  public BillableUsageRemittanceEntity getLatestRemittance(BillableUsage billableUsage) {
+  public double getTotalRemitted(BillableUsage billableUsage) {
     BillableUsageRemittanceEntityPK key = BillableUsageRemittanceEntityPK.keyFrom(billableUsage);
-    return billableUsageRemittanceRepository
-        .findById(key)
-        .orElse(BillableUsageRemittanceEntity.builder().key(key).remittedValue(0.0).build());
+    var filter =
+        BillableUsageRemittanceFilter.builder()
+            .orgId(key.getOrgId())
+            .billingAccountId(key.getBillingAccountId())
+            .billingProvider(key.getBillingProvider())
+            .accumulationPeriod(key.getAccumulationPeriod())
+            .metricId(key.getMetricId())
+            .productId(key.getProductId())
```

Comment:
Since remittance will always be calculated on an hourly basis, when determining the total remitted, we should be filtering by HOURLY only.

Reply:
ACK